File: board_dsl/src/main/java/com/dooho/board/api/user/MyPageDto.java
package com.dooho.board.api.user;

import com.dooho.board.api.board.BoardEntity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class MyPageDto {
    private String userEmail;
    private String userNickname;
    private String userProfile;
    private List<BoardEntity> userBoard;

}

File: board_spring/src/main/java/com/dooho/board/api/user/MyPageDto.kt
package com.dooho.board.api.user

import com.dooho.board.api.board.BoardEntity


data class MyPageDto (
    val userEmail: String? = null,
    val userNickname: String? = null,
    val userProfile: String? = null,
    val userBoard: List<BoardEntity?>? = null
    )